examples/win-title

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
`#!/usr/bin/env sh` is marginally more portable, but this is consistent with the others. I'll send a PR after this is merged.